assets/scss/admin/deactivation-feedback.scss

```diff
@@ -31,8 +33,9 @@
 			
 			li.reason {
 				margin-bottom: 10px;
-				
-				.reason-input {
+
+
+				.reason-input, .extra-message {
```

Comment:
@fajardoleo `extra-message` is not descriptive enough. Use `internal-message` instead.